content/test-content/docs-section-1/_index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Section 1 index"
+linkTitle: "Section 1 index and `linkTitle` Sample"
```

Comment:
The [Deploy preview](https://deploy-preview-39--cncf-hugo-starter.netlify.app/test-content/docs-section-1/) for this page doesn't seem to work correctly anymore?